optimum/exporters/onnx/model_patcher.py

```diff
@@ -239,7 +283,7 @@ def patched_forward(*args, **kwargs):
             # contains the output names of the model. In the case of Timm classification models, the output
             # is of type tensor. By default, it is assumed that the output names mentioned in the ONNX config
             # match the outputs in order.
-            filterd_outputs = {}
+            filtered_outputs = {}
```

Comment:
nice catch ! I'm embarrassed by the amount of times I've modified this file without seeing this x)